ledger/light/pulsemanager/pulsemanager.go

```diff
@@ -556,6 +529,63 @@ func (m *PulseManager) setUnderGilSection(
 	return jets, oldPulse, prevPN, nil
 }
 
+func (m *PulseManager) splitJets(ctx context.Context, jets []jetInfo, previous, current, new insolar.PulseNumber) ([]jetInfo, error) {
+	me := m.JetCoordinator.Me()
+	logger := inslogger.FromContext(ctx).WithFields(map[string]interface{}{
+		"current_pulse": current,
+		"new_pulse":     new,
+	})
+
+	for i, jet := range jets {
+		drop, err := m.DropAccessor.ForPulse(ctx, jet.id, previous)
+		if err != nil {
+			continue
```

Comment:
Log the error, please.

Reply:
extracted into 'hasSplitIntention' function with log